coalib/coala_main.py

```diff
@@ -29,7 +29,7 @@ def do_nothing(*args):
 
 
 STR_ENTER_NUMBER = 'Enter number (Ctrl-{} to exit): '.format(
-    'Z' if platform.system() == 'Windows' else 'D')
+    'Z-Enter' if platform.system() == 'Windows' else 'D')
```

Comment:
@guilherme-p @yzgyyang is there no way to exit the program in a single combination key stroke?

Reply:
Since we need to press enter after Ctrl-Z to terminate the program, the message shouldn’t be ‘Ctrl-Z-Enter’ since it may confuse people to press the three keys as a combination. Also, I think we could consider use Ctrl-C in this case, although it doesn’t send an EOF to terminate the program, coala should behave the same in both cases.